src/types/callable/function.js

```diff
@@ -26,8 +26,7 @@ export function createFunction(info) {
     );
 
     if (!success) {
-      console.error(`Error invoking function ${getName(info)}`);
-      return;
+      throw createGError(error);
```

Comment:
`createGError` should also handle when there is no error, and log that an error happened while trying to invoke the function, as before.